content/news/2024-02-13-fosdem-devroom.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Recordings of the FOSDEM Railways and Open Transport devroom available
+date: 2024-02-09
+image:
+  src: images/news/fosdem-2024-devroom.jpg
+  alt: Picture of the crowded devroom in 2024
+  class: image-position-center
+---
+
+For the second time, team members of the OpenRail Association have coordinated the **Railways and Open Transport devroom** at FOSDEM. For this 4-hours track we chose 10 presentations by experts with diverse backgrounds from the large amount of submissions.
+
+While most talks at FOSDEM circulated around programming, cloud, or system administration topics, we experienced an overwhelming interest in railways and transportation. Our devroom was constantly overcrowded with around 100 participants, with many interested people still standing outside who couldn't get in the room any more.
+
+### How Open Source is being used from planning to operations
+
+The talks in the devroom were following a consistent story: we started with a primer into the open standards and open data that powers the sector. We then looked at data to model pessenger data, followed by simulations and building timetables. Afterwards, we've seen examples how all this information is being used in the day-to-day operations in companies and community projects. Via the respective links you can find the video recording and slides of each talk, so please enjoy these great presentations:
```

Comment:
```suggestion
The talks in the devroom were following a consistent story: we started with a primer into the open standards and open data that powers the sector. We then looked at data to model passenger data, followed by simulations and building timetables. Afterwards, we've seen examples how all this information is being used in the day-to-day operations in companies and community projects. Via the respective links you can find the video recording and slides of each talk, so please enjoy these great presentations:
```

Reply:
Thanks @SergeCroise for contributing and fixing!

@SimonClavier, "information" is always singular in English ;)